### ISSUE TITLE
Скласти додаткові інформаційні блоки

### ISSUE THREAD

**Issue by bas-sergii** [bas-sergii/homepage#7]
Можна зробити резюме менш формальним, додавши додаткові блоки, які містять корисну інформацію, але подають її в "розважальній" формі. Наприклад, що подобається і не подобається, сильні і слабкі сторони, перелік навиків чи хобі, інфографіка або ж інтерактивні модулі, тощо. Варто написати контент такого блоку раніше, ніж починати верстку, бо його розміри впливатимуть на розміщення елементів на сторінці.

**Comment by SergiiKurinnyi** [comment]
Since I was a child I am interested both in computers and English, but unfortunately, I didn’t like math that much. My parents were against colleges and insisted on a University degree. That is why, after high school in the year 2001, I have chosen to apply to a University to English Language program instead of Computer Science. But I was addicted to computers, I played games, created basic websites etc. In the year 2005, before graduation, I started my own business – an online store selling computers and accessories. One year later, before getting a bachelor's degree, because of misunderstanding I was expelled from the University, before the final exam and writing diploma work (ironically, it was about the Canadian Education system). That was the biggest mistake in my life. But on the other hand, I didn’t (and I still don’t) see myself as a teacher and my business was rather successful. Our earnings were much higher and couldn't be compared to teachers’ salaries. But over the years the margin in sales began to shrink and that was the time to change the field of business. Additionally, Russia's occupation of Ukrainian territories affected the country’s economy a lot. 

In the year 2014, we have decided to move to Slovenia as business immigrants. Firstly, we planned to start with some basic IT services, but we didn’t have enough experience and at the same time, I found that the hospitality industry was rather profitable and rising in Slovenia. That’s how we decided to invest in accommodation service and I became an accommodation manager. In five years, at the beginning of 2019 accommodation prices reached enormous numbers and at the same time, people started talking about another upcoming economical crisis. As I was sure that the hospitality industry is the most vulnerable and travelling is the first thing people cut expenses on, so I felt that we should sell the company (that was one year before the COVID-19 pandemic) and so we did. By that time I still didn’t have a plan for my future career or another business, so to broaden my experience I decided to try myself as a truck driver. Rather soon I found this industry too hard to work at and too competitive to start a business in.

Very soon the pandemic started, which has confirmed my previous opinion about the hospitality industry and I didn’t proceed with an idea of studying in this direction. Also, I thought that if 2 persons can infect thousands of people, thousands can infect millions and this will last for long and that it is not the right time to start any kind of business. Therefore, it is the perfect time to change my life, make my mother’s dream come true and correct my mistakes from the past. As Steve Jobs said, “We have to work not 12 hours, and head”. That is how the idea of investing in my own education came up.

I applied to the University in my native city in Ukraine where it was possible to continue with a program similar to the one that I had before and learn remotely during the pandemic. According to the course plan, I should get a bachelor’s degree in June 2021. I found that learning appeals to me much more than it was 15 years ago. But still, I don’t see myself as a teacher and the future of the hospitality industry is uncertain, moreover, salaries in both industries are among the lowest. Therefore, this study doesn’t change much except of the fact that I will have a degree. My old passion, computer programming came into my mind. Also, I like the idea that many IT jobs are well-paid and at the same time allow people to work remotely from anywhere in the world. And I applied to programming online courses, which is good both for building a career or starting a new business. I keen on programming, but unfortunately, I don’t have commercial experience in this area and even after 7 months of studies I feel a lack of fundamental knowledge (as well as presentation skills) required to get a job.
